test/fixtures/README.md

```diff
@@ -0,0 +1,89 @@
+# Fixtures Directory
+
+This directory contains test fixtures for various certificate types and schema formats used across the Material Identity ecosystem. These fixtures serve as reference examples for implementation, testing, and validation purposes.
+
+## Directory Structure
+
+Fixtures are organized by certificate type and schema version:
+
+```
+fixtures/
+├── README.md                       # This file
+├── coa/                            # Certificate of Analysis fixtures
+│   ├── v0.x.y/                     # Version-specific fixtures
+│   │   ├── example_001.json        # Example certificate in JSON format
```

Comment:
I may be wrong but I think they have to be in the format `vallid_certificate_<number>.json` and same for PDF.

Reply:
Yes